docs: note on trigger syntax changes (#810)
Bearer/bearer: docs/guides/custom-rule.md

  description: "This is an example rule created based on the tutorial."
```

## Syntax updates
### v1.1 Trigger changes
If you have created a custom rule before v1.1 you will need to make the some small changes

#### Local, Present

> If you use `trigger: local` or `trigger: present` you can simply remove the trigger attribute and your rule should work as before

Reviewer: ```suggestion
If you use `trigger: local` or `trigger: present` you can simply remove the trigger attribute and your rule should work as before.
```

#### Absence

> If you have `trigger: absence`
> Replace it with the following syntax, and remove `trigger_rule_on_presence_of`

Reviewer: ```suggestion
If you use `trigger: absence`, replace it with the following syntax and remove `trigger_rule_on_presence_of` from your existing rule.
```

> ```

Reviewer: ```suggestion
```yaml
```

trigger:
  match_on: absence
  required_detection: # whatever value you had for `trigger_rule_on_presence_of`
```

#### Global

> for `trigger: global` replace it with the following syntax

Reviewer: ```suggestion
For `trigger: global` replace it with the following syntax.
```

> ```

Reviewer: ```suggestion
```yaml
```
